packages/safe-core-sdk/src/EthersSafe.ts

```diff
@@ -255,6 +255,22 @@ class EthersSafe implements Safe {
     return new SafeTransaction(standardizedTransaction)
   }
 
+  /**
+   * Returns a Safe transaction ready to be signed by the owners that invalidates a pending Safe transaction.
+   *
+   * @param safeTransaction - The pending Safe transaction to be invalidated
+   * @returns The Safe transaction that invalidates the pending Safe transaction
+   */
+  async rejectTransaction(safeTransaction: SafeTransaction): Promise<SafeTransaction> {
```

Comment:
I wonder if the name should be `createRejectionTransaction`. This fits more to the existing `createTransaction` and in my opinion closer to what is happening.

Reply:
Maybe in the SDK, we'd also want API to be about replacing a transaction. In the UI it was simplified to "cancelation", but maybe in the SDK, we can be more realistic about what's happening behind the scenes. The API can be way simpler too: `.replaceTransaction(nonce)`

Reply:
I like `createRejectionTransaction(nonce)`. Using the word "replace" is not my favourite option because it is true that we are replacing a transaction, but always with an empty transaction, not a custom one.